app/templates/partials/question.html

```diff
@@ -25,12 +26,18 @@ <h3 class="question__subtitle mars">{{question_subtitle}}</h3>
 
 {% set question_description %}
   {%- if question.description -%}
-    <div class="question__description mars" id="description-{{question.id}}">
+    <div class="question__description mars">
       {{question.description}}
+    {% for content in question.content %}
+        <div class="question__description mars">
```

Comment:
I think it might be more semantic to iterate some `<p>` tags within the description `<div>`. Each line of the description belongs to the description, it isn't a separate description.

E.g.
```
<div class="question__description mars">
  {{question.description}}
  {% for content in question.content %}
    <p>{{content.description}}</p>
  {% endfor %}
</div>
```